events/importer/kulke.py

```diff
@@ -31,6 +34,14 @@
 # Per module logger
 logger = logging.getLogger(__name__)
 
+
+EVENT_API_BASE_URL = os.getenv("EVENT_API_BASE_URL", "http://elis.helsinki1.hki.local/event-api/")
```

Comment:
Other env-based configs in this project go through `settings.py` so this one should as well. 

Also, as the env variable is global, it would be good if it was named a bit more specifically to Elis/Kulke (there is already a lot of "event" stuff in this project 😄). Maybe `ELIS_EVENT_API_BASE_URL` ?

Reply:
👍 Moved to settings and named it `ELIS_EVENT_API_URL`, which seems to be in line with other URLs there.